tools/mdanalysis/macros.xml

```diff
@@ -6,6 +6,7 @@
             <requirement type="package" version="1.2.1">scipy</requirement>
             <requirement type="package" version="0.9.0">seaborn</requirement>
             <requirement type="package" version="1.0.5">nbdime</requirement>
+            <requirement type="package" version="0.24.2">pandas</requirement>
```

Comment:
@tsenapathi this is not a good idea. Please include here `<yield/>` and put this dependency into the tool that needs it. See here: https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tools/freebayes/freebayes.xml#L8

the same you can do with citations, so only relevant citations are show even if you use macros.

Reply:
@bgruening I am extremely sorry for this mistake. Thank you for pointing it out! I have corrected it now.

Reply:
No need to be sorry. Was just a hint that this is possible :)